### PR TITLE
Added Button

### PR DESCRIPTION
**Pull request by kittygingell**
added a button, experimented with react state

### CODE REVIEW

File: src/NasaParagraph2.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react';
+export default class NasaParagraph2 extends Component {
```
Comment (rickythedeveloper) [inline src/NasaParagraph2.tsx:2]: Can you make this NasaParagraph

**Review by rickythedeveloper — CHANGES_REQUESTED** [review]
Added some comments pls fix

File: src/NasaParagraph2.tsx (hunk 1 above)
Comment (kittygingell) [reply src/NasaParagraph2.tsx:2]: We already have a NasaParagraph

Comment (rickythedeveloper) [reply src/NasaParagraph2.tsx:2]: cool

**Review by rickythedeveloper — APPROVED** [review]

**Pull request merged**